### ISSUE TITLE
DevTools loses connection after Hibernation 

### ISSUE THREAD

**Issue by feinstein** [flutter/devtools#2608]
After my Windows 10 computer wakes up from a hibernation I get this error:

![161063794932812226282906927752](https://user-images.githubusercontent.com/8120472/104611247-ad0d1980-5663-11eb-8113-ad0165fb3ede.jpg) 

Hot reload/restart still works, but not the DevTools.

This computer belongs to my employer, a bank, so there are multiple layers of networking security. This could be related, but it doesn't appear to affect hot reload though, just the DevTools.

**Comment by jacob314** [comment]
Does the Intellij debugger work as well? It would be helpful to know that the Java based version of the vm service client is still working after a resume but the Dart based version isn't.

**Comment by feinstein** [comment]
Yes, the debbuger works fine. I can click a button on the emulator and have a breakpoint triggered.

**Comment by elliette** [comment]
I'm able to reproduce this on macos and VS Code. 

Steps to reproduce:
- launch a Flutter app from VS Code (DevTools inspector panel automatically opens)
- click mac menu button > "Sleep"
- wake computer from sleep, go to your VS Code instance
- DevTools is now disconnected

**Comment by JohnCido** [comment]
Is this planned to be fixed? It's quite frustrating that I have to re-deploy the whole app after I leave my desk to grab a cup of water.
